controllers/cluster_controller_test.go

```diff
@@ -268,6 +267,34 @@ var _ = Describe(string(fdbtypes.ProcessClassClusterController), func() {
 					FullReplication:      true,
 					DataMovementPriority: 0,
 				}))
+
+				Expect(cluster.Status.StorageServersPerDisk).To(Equal([]int{1}))
+				Expect(cluster.Status.ImageTypes).To(Equal([]string{"split"}))
+			})
+		})
+
+		When("converting a cluster to use unified images", func() {
+			BeforeEach(func() {
+				cluster.Spec.UseUnifiedImage = pointer.Bool(true)
+				err = k8sClient.Update(context.TODO(), cluster)
+				Expect(err).NotTo(HaveOccurred())
+			})
+
+			It("should update the pods", func() {
+				pods := &corev1.PodList{}
+				err = k8sClient.List(context.TODO(), pods, getListOptions(cluster)...)
+				Expect(len(pods.Items)).To(Equal(len(originalPods.Items)))
+
+				for _, pod := range pods.Items {
+					Expect(pod.Spec.Containers[0].Name).To(Equal("foundationdb"))
+					Expect(pod.Spec.Containers[0].Image).To(Equal(fmt.Sprintf("foundationdb/foundationdb-kubernetes:%s", fdbtypes.Versions.Default)))
+					Expect(pod.Spec.Containers[1].Name).To(Equal("foundationdb-kubernetes-sidecar"))
+					Expect(pod.Spec.Containers[1].Image).To(Equal(fmt.Sprintf("foundationdb/foundationdb-kubernetes:%s", fdbtypes.Versions.Default)))
```

Comment:
This image doesn't exist currently right?

Reply:
That's right, the image has not been published. I've been testing it with a locally-built image.